src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13082,6 +13113,11 @@ void VRFTerminalUnitEquipment::CalcVRF_FluidTCtrl(EnergyPlusData &state,
         }
     }
     // calculate sensible load met using delta enthalpy
+    auto const &thisDXCoil = state.dataDXCoils->DXCoil(this->HeatCoilIndex);
+    // yujie: correct mismatching in the air mass flow rate
+    if (thisDXCoil.TotalHeatingEnergyRate > 0.0) {
+        AirMassFlow = state.dataDXCoils->DXCoil(this->HeatCoilIndex).InletAirMassFlowRate;
```

Comment:
`thisDXCoil` since you just defined it? [FWIW I am not sure what `this` is adding to `thisDXCoil`. What other DX coil is there here?]